### PR TITLE
MOP DS - Public Event Planning 

### PR DESCRIPTION
**Pull request by Samiha-Haque**
Hi @HarleyNgo and @RhuthK 
I have completed my usecase and created this pull request for review.

### CODE REVIEW

**Comment by HarleyNgo** [comment]
Well done!
A few things can be improved in my opinion:
- The training/testing epochs part is too long -> how about hide it because you have the results graphs so no one would actually read it.
- The overall outlines of the usecase is hard to follow -> you can change the outlines again and add more description of why you did that. For example, separate in 4 parts: Data Cleaning (for all 3 datasets), Answering scenario 1, Answering scenario 2, Answering scenario 3, etc... or why and how you chose FFNN...
- Put "pip install..." in comment to reduce to length of usecase

**Review by HarleyNgo — APPROVED** [review]

**Comment by Samiha-Haque** [comment]
Hi @HarleyNgo !
I hid the pip install outputs and the grid search outputs. I also linked each major code block at the top sections with the titles and description, i.e now you can read what is going on in each section and navigate by clicking the links to view the code blocks.

**Comment by Samiha-Haque** [comment]
Hi @HarleyNgo 
I included a short description of FFNN as a summary and at the very end I included a findings text. Let me know if it is okay to merge since the pull request was approved beforehand.
